includes/Integrations/Events_Calendar/Handlers/Event_Handler.php

```
@@ -0,0 +1,1288 @@
+        // Ensure timezone is set
+        if ( empty( $args['EventTimezone'] ) ) {
+            $args['EventTimezone'] = wp_timezone_string();
+        }
+
+        // Ensure all-day event flag is set
+        if ( ! isset( $args['EventAllDay'] ) ) {
+            $args['EventAllDay'] = 'no';
+        }
+
+        return $args;
+    }
```

PHPMD (2.15.0)

Avoid unused private methods such as 'ensure_required_fields'. (Unused Code Rules)

(UnusedPrivateMethod)